test/verify/check-metrics

```diff
@@ -1318,11 +1318,14 @@ class TestMetricsPackages(packagelib.PackageCase):
 
         redis_service = redisService(m.image)
         redis_package = "valkey" if redis_service == "valkey" else "redis"
-        extra_packages = ""
+        extra_packages = []
         if redis_package == "valkey" and m.image.startswith("fedora"):
             # Fedora has a compat package that we need to
             # remove at the same time as valkey.
-            extra_packages = "valkey-compat-redis"
+            extra_packages += ["valkey-compat-redis"]
+            # if redis_package == "valkey" and m.image.startswith("fedora-43"):
+            # Add packages that were split out from valkey
+            extra_packages += ["valkey-rdma", "valkey-tls"]
```

Comment:
```
❱❱ sudo rpm --erase --verbose bogus
error: package bogus is not installed
```
and exits with 1. So this may require the actual distro check. That also applies to later Fedoras, so that will have to be refined. Perhaps running `rpm -qa 'valkey*'`?

Reply:
Very true. Added `rpm --qf '%{NAME} ' -qa valkey-*` to get the extra packages in one line now. PTAL